### PR TITLE
Fix warm home discount bug

### PR DESCRIPTION
**Pull request by leenagupte**
Trello card: https://trello.com/c/oNzNLYSM

## Factcheck
[Energy grants calculator](https://smart-answers-pr-2665.herokuapp.com/energy-grants-calculator/y/help_with_fuel_bill/benefits%2Cproperty/1950-01-01/child_tax_credit%2Cesa%2Cincome_support%2Cjsa%2Cpension_credit%2Cworking_tax_credit/child_under_16)

## Expected Changes

[URL on GOV.UK](https://www.gov.uk/energy-grants-calculator/y/help_with_fuel_bill/benefits%2Cproperty/1950-01-01/child_tax_credit%2Cesa%2Cincome_support%2Cjsa%2Cpension_credit%2Cworking_tax_credit/child_under_16)

* Fix bug where no value is displayed for the warm home discount. Should show £140.

### Before

screen shot 2016-07-27 at 16 07 52

### After

screen shot 2016-07-27 at 16 09 40

### CODE REVIEW

**Comment by pmanrubia** [comment]
LGTM

File: lib/smart_answer_flows/energy-grants-calculator/outcomes/_warm_home_discount.govspeak.erb [hunk 1]
```diff
@@ -1 +1 @@
-[Warm Home Discount scheme](/the-warm-home-discount-scheme) - £<%= calculator.warm_home_discount_amount %> discount on your energy bills
+- [Warm Home Discount scheme](/the-warm-home-discount-scheme) - £140 discount on your energy bills
```
Comment (selfthinker) [inline lib/smart_answer_flows/energy-grants-calculator/outcomes/_warm_home_discount.govspeak.erb]: By moving the bullets inside the partials you undid the work I did in #2711. Not super important but is there a chance we could undo it?

Comment (leenagupte) [inline lib/smart_answer_flows/energy-grants-calculator/outcomes/_warm_home_discount.govspeak.erb]: @selfthinker Is that where that came from? Sorry. I didn't realise. #2711 Is probably going to be overwritten by all of the energy-grants-calculator changes as they come back from factcheck.

It's going to be easier to wait until they're all done, and then replay the changes.

Comment (selfthinker) [inline lib/smart_answer_flows/energy-grants-calculator/outcomes/_warm_home_discount.govspeak.erb]: Fair enough. @pmanrubia predicted list items like that will be difficult to keep.

**Pull request merged**